tools/rpm2img

```diff
@@ -408,18 +408,30 @@ for side in A B ; do
   fi
   dev="BOTTLEROCKET_DATA_${side}"
   dev="${!dev}"
-  mkfs.ext4 -d "${DATA_MOUNT}" "${dev}" "${partsize["DATA-${side}"]}M"
-  echo "${UNLABELED}" | debugfs -w -f - "${dev}"
+
+  # Only create the filesystem if the size is big enough to allocate enough
+  # inode-per-blocks for when we resize the fs to fit a bigger data partition.
+  # If we create the data fs now with the smaller size, the inode-per-block
+  # count will be capped and when we resize the filesystem to a larger size
+  # later, the total inode count will be capped based on the smaller initial
+  # filesystem creation size.
+  # If we don't create the filesystem now, we'll create the filesystem at
+  # runtime after the data partition has been resized.
+  if [ "${partsize["DATA-${side}"]}" -ge 1024 ]; then
+    kfs.ext4 -b 4096 -O resize_inode -E "resize=268435456" -i 16384 -I 256  -m 0 -d "${DATA_MOUNT}" "${dev}" "${partsize["DATA-${side}"]}M"
```

Comment:
I think we can just use the previous behavior here:
```suggestion
    mkfs.ext4 -m 0 -d "${DATA_MOUNT}" "${dev}" "${partsize["DATA-${side}"]}M"
```

since `mkfs.ext4` defaults will be fine for a large enough initial filesystem.